### PR TITLE
WQP Site Service

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: src/main/java/gov/usgs/owi/nldi/controllers/RestController.java [hunk 1]
```diff
@@ -1,94 +1,116 @@
...
-	public static final String DISTANCE = "distance";
+	public static final String DATA_SOURCE = "dataSource";
 
 	public static final String NAVIGATE = "navigate";
 	public static final String SESSION_ID = "sessionId";
+	public static final String COUNT_SUFFIX = "_count";
+	
+	public static final String HEADER_CONTENT_TYPE = "Content-Type";
+	public static final String MIME_TYPE_GEOJSON = "application/vnd.geo+json";
+    public static final String FEATURE_COUNT_HEADER = BaseDao.FEATURES + COUNT_SUFFIX;
```
Comment (mbucknell) [inline src/main/java/gov/usgs/owi/nldi/controllers/RestController.java]: There are a few indenting issues in this file.

**Comment by mbucknell** [comment]
Why are there a  lot of lines commented out in the pom.xml?

**Comment by mbucknell** [comment]
Otherwise looks good 👍

File: src/main/java/gov/usgs/owi/nldi/controllers/RestController.java (hunk 1 above)
Comment (dsteinich) [inline src/main/java/gov/usgs/owi/nldi/controllers/RestController.java]: Yuck, whitespace tabs vs. spaces is ugly.

**Pull request merged**